src/pages/videopage/videopage.js

```diff
@@ -1,17 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './videopage.css';
 import Videocard from '../../components/videocard/videocard';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
 
 export default function Videopage() {
+    const { _id } = useParams();
+    const [video, setVideo] = useState({});
+    const [videos, setVideos] = useState([]);
+    useEffect(async () => {
+        const res = await axios.get("/api/videos");
```

Comment:
Code suggestion:
Can destructure the object related to response and the ternary operator for shorter understanding. If it doesn't work you can shift to the same 
const { res:
  {data:
  {
videos:videoList
   }
}
}=await axios.get("/api/videos");
and then you can use `videoList` it in L13 and L14 
```
setVideos(videoList)
videoList.find(e=> e._id===id ? setVideo(e):"")
```

Reply:
Thanks for the suggestion @dev-enforced , tried implementing this but didn't work. Will read more about this and try again  in the next PRs.